inventory/routes/strategies/removal.rst

```diff
@@ -1,192 +1,136 @@
 ==================================================
-What is a removal strategy (FIFO, LIFO, and FEFO)?
+What is a Removal Strategy (FIFO, LIFO, and FEFO)?
 ==================================================
 
-Overview
-========
-
-Removal strategies are usually in picking operations to select the best
-products in order to optimize the distance for the worker, for quality
-control purpose or due to reason of product expiration.
+Removal strategies are a set of rules defined in picking operations that change which products
```

Comment:
I don't really like it... 

I don't think the distance to the worker matters that much. I wouldn't focus on that. 

I also think it's connected to Accounting, you should double-check with OMA. I don't think this intro works.

Reply:
Buenos dias, 

What would be the fate of this Pull Request?

Cheers :)